app/coffeescript/components/mixins/map_utils.coffee

```diff
@@ -3,12 +3,13 @@ define [ 'jquery' ], ( $ ) ->
   mapUtils = ->
     @defaultAttrs
       assetUrl: undefined
-      assetHostSelector:    'meta[name="asset_host"]'      
-      spinnerSelector:      '.spinner'
-      refinementsSelector:  '.pageInfo[name="refinements"]'
-      propertyNameParam:    'propertyname'
-      mgtcoidParam:         'mgtcoid'
-      propertyManagementRE: 'property-management'
+      assetHostSelector:      'meta[name="asset_host"]'
+      spinnerSelector:        '.spinner'
+      refinementsSelector:    '.pageInfo[name="refinements"]'
+      pinRefinementsSelector: '.pageInfo[name="pin_search_refinements"]'
```

Comment:
Here are some examples:

URL: http://local.apartmentguide.com/map?state=Georgia&city=Atlanta&min_price=1300&max_price=1500
```javascript
$('.pageInfo[name="refinements"]')
[]
$('.pageInfo[name="pin_search_refinements"]')
[<meta class=​"pageInfo" content=​"max_price=1500&min_price=1300" name=​"pin_search_refinements">​]
```

This is the problem use case. We can't just use the existing `refinements` element from the DOM because we need the min & max price data to pass to the info_window.

http://local.apartmentguide.com/map?state=Georgia&city=Atlanta&refinements=2-beds-1z141y8&min_price=1300&max_price=1500

```javascript
$('.pageInfo[name="pin_search_refinements"]')
[<meta class=​"pageInfo" content=​"refinements=2-beds-1z141y8&max_price=1500&min_price=1300" name=​"pin_search_refinements">​]
$('.pageInfo[name="refinements"]')
[<meta class=​"pageInfo" content=​"2-beds-1z141y8" name=​"refinements">​]
```

Note that the `refinements` element does not include the "refinements=" string, because map.js is going to add it.

Hope this is clear. Basically we don't want to try to make this one field do both things because of all the dependencies from both AG and Rent. Happy to discuss further.